docusaurus/docs/learn/quickstarts/network/local-no-docker.mdx

```diff
@@ -6,7 +6,7 @@ sidebar_position: 30
 import Wizardly from '@site/src/components/Wizardly';
 
 :::info
-Quickstarts are short-lived networks that are great for learning how to use OpenZiti. For long-lived production deployments, see [the deployment guides](/docs/category/deployments).
+Quickstarts are short-lived networks that are great for learning how to use OpenZiti. For long-lived production deployments, see [the deployment guides](@openzitidocs/category/deployments).

```

Comment:
there are some docusaurus links that are unpossible to reference because theya re synthesized. links to "category" urls must use the alias